packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -96,6 +96,9 @@ install -m 0644 .%{gem_instdir}/config/%{plugin_name}.yml \
 %{gem_instdir}/test
 
 %changelog
+* Wed Sep 02 2020 Chris Roberts - 0.23-0.2.pre.master
```

Comment:
I think the prerelease stuff needs to be removed; check the 2.1 branch to see what was done there